README.md

```diff
@@ -1,64 +1,70 @@
-# The Poseidon Standard
+## The Poseidon Standard v2.7.1
 
-Poseidon is a solution for genotype data organisation established within the Department of Archaeogenetics at the Max Planck Institute for the Science of Human History (MPI-SHH) in Jena.
+Poseidon is a solution for archaeogenetic genotype data organisation.
 
-Detailed documentation for Poseidon can be found at [https://poseidon-framework.github.io](https://poseidon-framework.github.io)
+This standard defines the core components of the Poseidon package. Further details on [genotype data](https://poseidon-framework.github.io/#/genotype_data), the [.janno file](https://poseidon-framework.github.io/#/janno_details) and the [.ssf file](https://poseidon-framework.github.io/#/ssf_details) are documented on the Poseidon website.
 
-A changelog is available [here](https://poseidon-framework.github.io/#/changelog).
+A changelog for this standard is available on the website [here](https://poseidon-framework.github.io/#/changelog).
 
-## The Poseidon package structure
+### The Poseidon package structure
 
-All ancient and modern data in Poseidon are distributed into so-called packages, which are directories containing a dedicated set of files. Packages correspond to published sets of genomes, or in case of unpublished projects, ongoing (and growing) sets of samples currently analysed. All text files in the package are UTF-8 encoded.
+A Poseidon package stores genotype data with context information for DNA samples from (ancient) (human) individuals. Packages are defined by the POSEIDON.yml file, which holds relative paths to all other files in a package.
 
-Every package should have the following files: 
+A package therefore must contain:
```

Comment:
See my comment above. I would all-caps the keywords `MUST`, `CAN/COULD`, `SHOULD`